spyder_kernels/py3compat.py

```diff
@@ -24,6 +24,8 @@
 import os
 import sys
 
+from ipython_genutils.py3compat import input
```

Comment:
Please copy the code that declares `input` in `ipython_genutils.py3compat` here. That way we'll avoid having to depend on that package.